data/anthonrodgrs01.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Anthon Rodrigues",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Developer, Problem solver, Getting started with Open source",
+  "avatar": "https://github.com/anthonrodgrs01.png",
```

Comment:
In our recent update, we no longer need the `avatar` key. So please remove this from your json file. 
```suggestion
```